### DISCUSSION TITLE
After update e generate new admin token still get the message

### DISCUSSION THREAD

**Discussion by riccardospeggiorin-centropaghe** [dani-garcia/vaultwarden#3581] [Q&A]
I updated to the lastest version and I receive the message that I am using a plain text admin_token in the environments variables.
I follow the instruction, run the temporary container and generated the argon2 string from my original admin_token. I then escape the dollar signs and put on the docker compose file in the place of the old one.
Still getting the message 

[NOTICE] You are using a plain text `ADMIN_TOKEN` which is insecure.
Please generate a secure Argon2 PHC string by using `vaultwarden hash` or `argon2`.

What I am missing? thanks

**Comment by riccardospeggiorin-centropaghe** [comment]
never mind...I found the solution...the config.json overwrite the env variables...so I update the token with the argon2 string from the admin panel.
thanks

**Reply by datzent83** [reply]
How did you manage to update the token with the argon2 string from the admin panel? I can't find that anywhere in the admin panel.
